From a30409ac88488758b36f624dc79e7a017432f192 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 02 Dec 2021 13:20:34 +0200
Subject: [PATCH] [SW-0] gaudi2: plls: fix comment

The actual PCI PLL frequency configuration is 500MHz, not 1600MHz.
Fix the comment.

Change-Id: Ia28599fdfcdbf050954fb0716e4e6c3956d5c2bb
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi2/common/asic.c b/arch/gaudi2/common/asic.c
index 9fbe159..2fb8162 100644
--- a/arch/gaudi2/common/asic.c
+++ b/arch/gaudi2/common/asic.c
@@ -105,7 +105,7 @@
 	{
 		.name = "PCI_PLL",
 		.base = mmDCORE0_PCI_PLL_CTRL_BASE,
-		.nr = 1, .nf = 60, .od = 3, .nb = 2,	/* 1600 */
+		.nr = 1, .nf = 60, .od = 3, .nb = 2,	/* 500 */
 		.div_factor = { -1, -1, 4, 4 },
 		.div_sel = { HL_PLL_PLL_CLK,
 			     HL_PLL_PLL_CLK,
